### DISCUSSION TITLE
Feature request: An option to not start authentication right away

### DISCUSSION THREAD

**Discussion by benny-noumena** [JoseGoncalves/vue-keycloak#10] [Ideas]
What about adding a flag and a way to start the authentication process when the page is rendered?

I think that would be a better UX.

Best

**Comment by JoseGoncalves** [comment]
I think that is achievable if you set `'check-sso'` in the `onLoad` prop of the `initOptions`object.
See: https://www.keycloak.org/docs/latest/securing_apps/#using-the-adapter
After that you will need to call the `login()` method to initiate the authentication.

**Comment by benny-noumena** [comment]
Oh yes, you are right. For me I had a interceptor that did the login for me automatically. Thank you.
